README.md

```diff
@@ -137,13 +137,55 @@ Based on the size and strucutre of the dataset provided + these parameter settin
 **_Note:_**: Not all models trained in studio will expose the `batch_size` parameter. For those that don't you should ensure your `max_batch_size_train` is larger than the default batch size (generally 16).
 </details>
 
-### Flags
+## Output
+The `output_path` should be a directory that will contain all the tokenized HDF5 split files, and a directory called `tokenizer`. This directory constitutes a processed dataset and can be used for training a model after uploading to SambaStudio. The `tokenizer` directory will be transferred to any output checkpoints that are saved by Sambastudio for the tokenizer to be used for inference later on.
```

Comment:
", and a directory called `tokenizer`.This directory constitutes a processed..." 

This working implies that the second sentence is referring to tokenizer directory, when in reality is refering to output_path directory